databroker/cli.py

```diff
@@ -17,6 +17,23 @@
 )
 
 
+def parse_dict_arg(arg):
+    """
+    Parse a string like '{"key1": "value1", "key2": 2}' into a dictionary.
+
+    Returns:
+        dict: The parsed dictionary or an empty dict if arg is None or empty.
+    """
+    if arg is None or arg.strip() == '':
+        return {}
+
+    result = ast.literal_eval(arg)
```

Comment:
Using a json parser is a bit safer and more mainstream here.

Reply:
Good point. Squared away in https://github.com/bluesky/databroker/pull/847/commits/fb8c69f9698a7abe5831ffdc67fe43ce6ff17bad